### DISCUSSION TITLE
How to Get Support

### DISCUSSION THREAD

**Discussion by benallfree** [pockethost/pockethost#401] [Announcements]
## System Health

If you think there is an issue with pockethost.io, please check https://status.pockethost.io and use [the uptime megathread](#223).

## General Support

General support is found here, in the Discussions area. If you're not sure where to begin, start here.

## Discord

Our [Discord server](https://discord.gg/nVTxCMEcGT) has over 1,000 members ready to help.

## Paid Support by Email

PocketHost is a #buildinpublic project, so using public support channels is preferred. However, if you are a paid member and are having trouble, you may email us.

**Comment by humbertomorais** [comment]
I subscribed to the Starter Plan and it was not updated on the Dashboard. My email is humberto.silva@ifma.edu.br. Could you help me?

**Comment by wasylmowczan** [comment]
@benallfree  Seems like an invitation link has expired

**Reply by benallfree** [reply]
Just tested it.

<img width="687" alt="image" src="https://github.com/user-attachments/assets/b374d982-5759-4ea9-be02-4ebce7e9f79d" />

**Comment by wasylmowczan** [comment]
![CleanShot 2025-03-05 at 17 00 37](https://github.com/user-attachments/assets/e2c56059-8a8d-417e-addf-6f91bbc39516)
This is what I see.

**Reply by benallfree** [reply]
What's your discord handle?

**Reply by wasylmowczan** [reply]
I figured it our, joint directly from Discord.

**Comment by thecolonykdt** [comment]
I subscribed to $5 per month and now I cannot login or verify my account , it keeps loading when I click verify

**Reply by benallfree** [reply]
Please email ben@pockethost.io or jump on Discord :)
